wire-ios-transport/Source/Background/BackgroundActivityFactory.swift

```diff
@@ -42,6 +40,8 @@ private let zmLog = ZMSLog(tag: "background-activity")
 
 @objc public final class BackgroundActivityFactory: NSObject {
 
+    private let logger = WireLogger(tag: "background-activity")
```

Comment:
we usually do WireLogger.backgroundActivity so there's a unique tag defined in WireLogger

Reply:
thanks changed